### PR TITLE
Fixing major code smells reported in sonar code analysis in commons module.

### PR DESCRIPTION
**Pull request by nagachandra-immadisetty**

### CODE REVIEW

File: commons/src/main/java/io/hyscale/commons/commands/CommandExecutor.java [hunk 1]
```diff
@@ -199,5 +194,6 @@ private static OutputStream getOutputStream(File outputFile) throws FileNotFound
         }
         return new StringOutputStream();
     }
-    
+
+    private CommandExecutor(){}
```
Comment (SameeraGrandhi) [inline commons/src/main/java/io/hyscale/commons/commands/CommandExecutor.java:198]: Should be at the beginning of the class.

File: commons/src/main/java/io/hyscale/commons/framework/patch/StrategicPatch.java [hunk 2]
```diff
@@ -233,4 +233,6 @@ private static HyscaleException getException(String key) {
         return ex;
     }
 
+    private StrategicPatch() {}
```
Comment (SameeraGrandhi) [inline commons/src/main/java/io/hyscale/commons/framework/patch/StrategicPatch.java:236]: Beginning of the class

File: commons/src/main/java/io/hyscale/commons/io/HyscaleFilesUtil.java [hunk 3]
```diff
@@ -278,5 +279,7 @@ public static String readFileData(File filepath) throws HyscaleException {
 	    
 	    return matchingFile == null ? null : Arrays.asList(matchingFile);
 	}
+
+	private HyscaleFilesUtil() {}
```
Comment (SameeraGrandhi) [inline commons/src/main/java/io/hyscale/commons/io/HyscaleFilesUtil.java:283]: Beginning of the class

File: commons/src/main/java/io/hyscale/commons/models/YAMLManifest.java [hunk 4]
```diff
@@ -19,14 +19,14 @@
 
 public class YAMLManifest implements Manifest {
 
-	private File yamlManifest;
+	private File manifestFile;
```
Comment (SameeraGrandhi) [inline commons/src/main/java/io/hyscale/commons/models/YAMLManifest.java:22]: It's already evident that it is a file with Data Type , so just manifest would suffice here.

File: commons/src/main/java/io/hyscale/commons/constants/K8SRuntimeConstants.java [hunk 5]
```diff
@@ -62,4 +62,6 @@
 	
 	public static final Integer DEFAULT_REPLICA_COUNT = 1;
 
+	private K8SRuntimeConstants() {}
```
Comment (SameeraGrandhi) [inline commons/src/main/java/io/hyscale/commons/constants/K8SRuntimeConstants.java:65]: Beginning of the class

File: commons/src/main/java/io/hyscale/commons/constants/ValidationConstants.java [hunk 6]
```diff
@@ -64,5 +64,6 @@
     public static final String STRUCTURED_OUTPUT_FORMAT_REGEX = "JSON|json";
 
     public static final String INVALID_OUTPUT_FORMAT_MSG = "Output format \"{}\" is invalid. Supported format is json.";
-    
+
+    private ValidationConstants(){}
```
Comment (SameeraGrandhi) [inline commons/src/main/java/io/hyscale/commons/constants/ValidationConstants.java:68]: Beginning

Comment (nagachandra-immadisetty) [reply commons/src/main/java/io/hyscale/commons/constants/ValidationConstants.java:68]: In All Constants file I kept the constructors at the end as per coding conventions.

**Pull request merged**